lib/install/data/data.go

```diff
@@ -263,5 +263,7 @@ func (d *Data) CopyNonEmpty(src *Data) error {
 
 	d.Timeout = src.Timeout
 
+	d.DNS = src.DNS
```

Comment:
IsSet is checked in copychangedconfig, so it's safe to copy over. DNS here is []net.IP so I can't check IsSet here